_includes/head.html

```diff
@@ -19,30 +19,30 @@
     <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
     <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
     <!-- Essential styles -->
-    <link rel="stylesheet" href="{{ site.path.plugins }}/bootstrap/css/bootstrap.min.css" type="text/css">
-    <link rel="stylesheet" href="{{ site.path.plugins }}/font-awesome/css/font-awesome.min.css" type="text/css">
-    <link rel="stylesheet" href="{{ site.path.plugins }}/fancybox/jquery.fancybox.css?v=2.1.5" media="screen">
+    <link rel="stylesheet" href="/assets/plugins/bootstrap/css/bootstrap.min.css" type="text/css">
```

Comment:
this should be a variable path with a prepend url like `{{ site.path.plugins | prepend: site.baseurl }}`